### PR TITLE
Create translation.json

### PR DESCRIPTION
**Pull request by vanja-san**
ru

### CODE REVIEW

**Comment by ryuzeen** [comment]
Linha 27 : "no_downloads_in_progress": "Скачиваний не выполняется", troque pra Скачивание не выполняется
Linha 105 : Добалвено -> Добавлено

**Comment by ryuzeen** [comment]
"featured": "Рекомендуемые", troca pra "Выделите"

**Comment by hydralauncher** [comment]
`catalogue.no_results` is missing

File: src/locales/ru/translation.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+  "catalogue": {
+    "featured": "Рекомендуемые",
+    "recently_added": "Новые",
+    "trending": "Популярные",
+    "surprise_me": "✨ Случайная"
```
Comment (GuizaumDEV) [inline src/locales/ru/translation.json:6]: "surprise_me": "✨ Случайная" é mais apropriado usar "удивите меня"

**Comment by GuizaumDEV** [comment]
"space_left_on_disk": "На диске осталось {{space}}", troca pra "свободное дисковое пространство"

**Comment by GuizaumDEV** [comment]
"no_minimum_requirements": "У {{title}} не предоставлено информации о минимальных требованиях", troca pra никаких минимальных требований

**Comment by GuizaumDEV** [comment]
linha 52 - нет рекомендуемых требований

**Comment by vanja-san** [comment]
@hydralauncher What's going on here? This is the first time that someone has found it so difficult to accept a translation, it’s strange. Should I send it as a *patch* or what?

**Comment by vanja-san** [comment]
@hydralauncher I got it. Can you please signal me when text is more-less ready, then I will do the translation. 🙂👌 You can automate process by adding English lines to all files, and users will translate them later. This way you won't have to wait.

**Comment by hydralauncher** [comment]
@vanja-san you can translate the latest [en file](https://github.com/hydralauncher/hydra/blob/main/src/locales/en/translation.json) to ru, as soon as we add new texts, we will add an automatic translation and you can open a pr to correct it

We released v1.1.0 today, so we won't be adding any new text for now (i guess)

**Pull request closed without merging**